tests/integration/utils/test_normalizers.py

```diff
@@ -23,6 +23,11 @@
 from __future__ import absolute_import, division, print_function
 
 from inspirehep.utils.normalizers import normalize_journal_title
+from inspirehep.modules.literaturesuggest import normalizers
+
+
+class MockObj:
+    pass
```

Comment:
There was this issue that this class was in the unit tests and It couldnt be imported